draft-add-requirements.md

```diff
@@ -32,21 +56,34 @@ informative:
 --- abstract
 
 Adaptive DNS Discovery is chartered to define mechanisms that allow clients to discover and select encrypted DNS resolvers.
-This document seeks to provide a set of use cases and requirements that proposed mechanisms should address.
+This document describes several use cases for discovering DNS resolvers that support encrypted transports,
+and lists requirements that any proposed discovery and selection mechanisms should address.
 
 --- middle
 
 # Introduction
 
-The Adaptive DNS Discovery (ADD) Working Group is chartered to define mechanisms that allow clients to discover and select
-encrypted DNS resolvers in a wide variety of network environments. This document seeks to provide a set of use cases and
-requirements that proposed mechanisms should address. They can do this either by providing a solution, or by
-explicitly stating why it is not in scope.
+Several protocols for protecting DNS traffic with encrypted transports have been defined,
+such as DNS-over-TLS (DoT) {{?RFC7858}} and DNS-over-HTTPS (DoH) {{?RFC8484}}.
```

Comment:
You may want to add DNS-over-QUIC to the list (https://tools.ietf.org/html/draft-ietf-dprive-dnsoquic-00).

Reply:
Yes we could, although I feel it would make the first sentence too long. This sentence should only really have two examples to illustrate what we mean. But perhaps somewhere further down we should define a list of which protocols should be supported by this discovery process?

Reply:
Works for me, we can add in the terminology section:
 'Encrypted DNS' refers to DNS-over-HTTPS [RFC8484], DNS-over-TLS [RFC7858], or DNS-over-QUIC [I-D.ietf-dprive-dnsoqu

Reply:
Since this is an example list, I'd prefer to leave it to protocols that are already deployed RFCs. Of course, the recommendations here apply to DoQ, but this isn't supposed to be exhaustive.

Reply:
DNS-over-QUIC is already a WG document and will most likely progress to publication before the work in ADD WG.

Reply:
Sure, I could go either way.

Reply:
We didn't have a Terminology section before but I've now committed one, including this Encrypted DNS definition and also associated/equivalent/alternative. I tried the reflect the format used in other RFCs so hope that looks ok. Resolving for now.